### PR TITLE
Profile privacy new2

### PR DESCRIPTION
**Pull request by hanlun0804**
Closes: #...

## Checklist

- [ ] I have written tests
- [ ] I have provided documentation

## Changelog

- A bullet list of what you've done to make the life of the reviewer easier.

## How to test

A consise description of how to test the newly implemented functionality.

### CODE REVIEW

**Review by henrikskog — APPROVED** [review]
Good job! 😊

File: apps/web/src/components/views/ProfileView/components/ProfileEmailPrimary.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu"
```
Comment (kharann) [inline apps/web/src/components/views/ProfileView/components/ProfileEmailPrimary.tsx:2]: This should be a sepperate component imo and probably a part of the UI package

**Review by kharann — CHANGES_REQUESTED** [review]

**Comment by kharann** [comment]
I also think that it has a lot of temporary code which changes once a backend is in place. Perhaps but this PR on ice until it's done so we won't forget

File: apps/web/src/components/views/ProfileView/components/ProfileEmailPrimary.tsx (hunk 1 above)
Comment (hanlun0804) [reply apps/web/src/components/views/ProfileView/components/ProfileEmailPrimary.tsx]: Email tab is removed anyways, removed the files now :)

**Review by henrikskog — APPROVED** [review]
Good job!

**Pull request merged**